pkg/hooks/info/api.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package kubevirt.hooks.info;
+
+service Info {
+  rpc Info (InfoParams) returns (InfoResult);
+}
+
+message InfoParams {}
+
+message InfoResult {
+  // name of the hook used by virt-launcher to compare it with requested hooks
+  string name = 1;
+  // hookPoints is a list of subscribed hook points
+  repeated HookPoint hookPoints = 3;
+  // versions is a list of implemented hook Callbacks service versions
+  repeated string versions = 4;
+}
+
+message HookPoint {
+  // name represents name of the subscribed hook point
+  string name = 1;
+  // priority is used to sort hooks prior to their execution (second key is the name)
+  int32 priority = 2;
```

Comment:
mark optional priority with default value of 0 - i.e.: `optional uint8 priority = 2 [default = 255 ]; ` - I guess in Go is no issue since all types have default assignment, but other langs will not.

Reply:
Good point. I want to keep priority signed though, default 0, user can go up or down.

Reply:
`api.proto:24:12: Explicit 'optional' labels are disallowed in the Proto3 syntax. To define 'optional' fields in Proto3, simply remove the 'optional' label, as fields are 'optional' by default.`

We are good.